src/content/dvt/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Distributed Validator Technology
+description: Distributed Validator Technology enables the distributed operation of an Ethereum validator by multiple parties.
+lang: en
+---
+
+# Distributed Validator Technology {#what-is-dvt}
+_With the advent of Ethreum’s PoS consensus, standard validator setups were accompanied by some challenges and the risk of stake centralization. To help distribute validator nodes and decentralize the network, Distributed Validator Technology was conceived._
+
+<Divider />
+
+## What is DVT? {#what-is-dvt}
+
+**Distributed Validator Technology (DVT)** is a technology primitive in the evolving **Ethereum technology** stack aimed at improving the resilience and health of the Ethereum blockchain. The solution focuses on strengthening the backbone of the blockchain's PoS consensus by enabling multiple non-trusting parties to operate an Ethereum validator collectively, as opposed to how standard validators are operated. This shift in validator operation dramatically improves the liveness and security of the network while significantly reducing slashing, downtime, and centralization risks compared to standard validator setups. 
+
+A way to think about DVT is that DVT is to validators as multisig is to wallets. Where standard validators use a single key to sign duties, the key of a Distributed Validator is split across multiple nodes. In the case of DVT, the "multisig" is operated by a consensus protocol (BFT) instead of human interaction. Enabling the distributed operation of an Ethereum validator with best-in-class security practices. 
+
+Distributing a new or existing validator is quite complex in practice, but using a DVT solution like ssv.network as a layer to split the validator private key into multiple key shares makes the technology more accessible. Regardless of the [DVT-implementation](<https://ssv.network/>) stakers can have multiple nodes each operating a key share in a cluster, allowing them to ensure the uptime of their validators.
+
+## How does DVT work? {#how-does-dvt-work}
+
+DVT uses multiple components to facilitate the distribution of an Ethereum validator:
+
+![A Diagram showing how a single validator key is split into key shares and distributed to multiple nodes with varying components.](./dvt-cluster.png)
+
+- **[Shamir's Secret Sharing](https://medium.com/@keylesstech/a-beginners-guide-to-shamir-s-secret-sharing-e864efbf3648)** - Validator keys use BLS signatures - that are additive - allowing multiple signatures to be combined to recreate a validator key signature. The BLS private validator key is split into multiple "key shares." Using [BLS aggregation](https://our.status.im/fastest-bls-signature-implementation/), multiple shares or signatures can be aggregated into a single signature. 
```

Comment:
```suggestion
- **[Shamir's Secret Sharing](https://medium.com/@keylesstech/a-beginners-guide-to-shamir-s-secret-sharing-e864efbf3648)** - Validators use BLS keys. Individual BLS keys can be combined into a single aggregate very efficiently. In DVT, the private key for a validator is the [aggregate](https://our.status.im/fastest-bls-signature-implementation/) of the BLS signatures of each operator in the cluster. 
```

Reply:
**[Shamir's Secret Sharing](https://medium.com/@keylesstech/a-beginners-guide-to-shamir-s-secret-sharing-e864efbf3648)** - Validators use [BLS keys](https://en.wikipedia.org/wiki/BLS_digital_signature). Individual BLS keys ("key shares") can be combined into a single aggregated key (signature). In DVT, the private key for a validator are the the [combined BLS signatures](https://our.status.im/fastest-bls-signature-implementation/) of each operator in the cluster.

Reply:
ok - using your text here, but with a typo correction
```suggestion
- **[Shamir's Secret Sharing](https://medium.com/@keylesstech/a-beginners-guide-to-shamir-s-secret-sharing-e864efbf3648)** - Validators use [BLS keys](https://en.wikipedia.org/wiki/BLS_digital_signature). Individual BLS keys ("key shares") can be combined into a single aggregated key (signature). In DVT, the private key for a validator is the combined BLS signature of each operator in the cluster.
```